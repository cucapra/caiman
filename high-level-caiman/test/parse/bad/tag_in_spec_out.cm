#version 0.1.0
val g(f: (i32,)) -> (i32,) @ output(v.x)-? {
    x :- 10
}