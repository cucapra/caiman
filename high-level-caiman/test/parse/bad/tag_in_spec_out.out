Error: At tag_in_spec_out.cm:2:28, 
  Parsing Error: Unrecognized token: '@', expected one of [
    "\")\"",
    "\",\"",
    "\"const\"",
    "\"extern\"",
    "\"feq\"",
    "\"fn\"",
    "\"pipeline\"",
    "\"sptl\"",
    "\"tmln\"",
    "\"type\"",
    "\"val\"",
    "\"{\"",
    "\"}\"",
]
